manifest.toml

```diff
@@ -81,7 +90,11 @@ ram.runtime = "2G"
     api.auth_header = false
 
     [resources.apt]
-    packages = "postgresql"
+    packages = "postgresql, ca-certificates, git, curl, tar"
+
+    extras.docker.repo = "https://download.docker.com/linux/debian bullseye stable"
```

Comment:
Should the `ynh_install_extra_app_dependencies` helper be called to install those extra packages ?